index.js

```diff
@@ -225,3 +225,5 @@ module.exports = fs => {
         hashElement: hashElement
```

Comment:
I would like to return `hashElement` instead of an object, this allows to get rid of the `Object.assign` in line 229